File: SDKs/zuora.com/2021-08-20/python/src/sdk/models/shared/getinvoicefilewrapper.py
import dataclasses
from typing import Any,Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class GetInvoiceFileWrapper:
    invoices: Optional[list[dict[str, Any]]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('invoices') }})
    next_page: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('nextPage') }})
    success: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('success') }})
    

File: SDKs/zuora.com/2021-08-20/typescript/src/sdk/models/shared/getinvoicefilewrapper.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetInvoiceFileWrapper extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=invoices" })
  invoices?: Record<string, any>[];

  @SpeakeasyMetadata({ data: "json, name=nextPage" })
  nextPage?: string;

  @SpeakeasyMetadata({ data: "json, name=success" })
  success?: boolean;
}
